Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -677,6 +672,60 @@ bool BluetoothRealDevice::OpenDevice(const libusb_device_descriptor& device_desc
   return true;
 }
 
+std::vector<BluetoothRealDevice::BluetoothDeviceInfo> BluetoothRealDevice::ListDevices()
+{
+  std::vector<BluetoothDeviceInfo> device_list;
+  LibusbUtils::Context context;
+
+  if (!context.IsValid())
+    return {};
+
+  int result = context.GetDeviceList([&device_list](libusb_device* device) {
+    libusb_device_descriptor desc;
+
+    if (libusb_get_device_descriptor(device, &desc) != LIBUSB_SUCCESS)
+      return true;
+
+    auto [config_ret, config] = LibusbUtils::MakeConfigDescriptor(device, 0);
+    if (config_ret != LIBUSB_SUCCESS)
+      return true;
+
+    for (uint8_t j = 0; j < config->bNumInterfaces; j++)
```

Comment:
I don't think you need to iterate the interfaces, you can rely on USB Bluetooth being on interface 0x00. You can look at `BluetoothRealDevice::Open`.